compiler/src/org.graalvm.compiler.core.common/src/org/graalvm/compiler/core/common/GraalOptions.java

```diff
@@ -94,7 +94,7 @@
     public static final OptionKey<Boolean> LoopPeeling = new OptionKey<>(true);
 
     @Option(help = "", type = OptionType.Debug)
-    public static final OptionKey<Boolean> ReassociateInvariants = new OptionKey<>(true);
+    public static final OptionKey<Boolean> Reassociation = new OptionKey<>(true);
```

Comment:
I think this name alone is not descriptive enough, maybe ReassociateLoopInvariants? Also, please add a small sentence in the help text above in the annotation

Reply:
`Reassociation` is not only used for loop invariants, but also the constant reassociation. So do you have any better idea for it? Thanks!

Reply:
> not only used for loop invariants, but also the constant reassociation
I know, but the important use case is loops, thus the suggestion. But you are right. I think the name of the phase is fine, though. Maybe sth like ReassociateExpressions?

Reply:
Yeah, make sense! I will change the name in the new patch. Thanks!